ui/css/style.css

```diff
@@ -747,6 +748,41 @@ a.button {
   .nav-search {
     margin-right: 15px; } }
 
+.rate {
+  float: left;
+  height: 46px;
+  padding: 7px 10px; }
+
+.rate:not(:checked) > input {
+  position: absolute;
+  top: -9999px; }
+
+.rate:not(:checked) > label {
+  float: right;
+  width: 1em;
+  overflow: hidden;
+  white-space: nowrap;
+  cursor: pointer;
+  font-size: 21px;
+  color: #ccc; }
+
+.rate:not(:checked) > label:before {
+  content: '★ '; }
+
+.rate > input:checked ~ label {
+  color: #ffc700; }
+
+.rate:not(:checked) > label:hover,
```

Comment:
@mcaleb808 Try to use nested sass feature effectively to have more specific selectors